### DISCUSSION TITLE
Elevate user to DJ if they are alone

### DISCUSSION THREAD

**Discussion by the-snesler** [jagrosh/MusicBot#832] [Ideas]
Hello! I have a small suggestion that would increase the quality of life for small servers: If a user is alone with the bot (or if there are no DJs currently in the voice channel?), then that user should be able to access settings such as pausing playback, changing volume, or looping the queue. For those who prefer the current system, this could be added as a config option.
Screenshots of the current system:
![image](https://user-images.githubusercontent.com/78718829/133690528-1383f8b7-40c6-4e0b-969a-3e6de646f814.png)
![image](https://user-images.githubusercontent.com/78718829/133690535-e340a7c2-2a79-498c-a3c6-ab015222d9e7.png)

**Comment by MichailiK** [comment]
There are some considerations when doing this. One predominant one is, what should happen if a second user joins?

**Comment by jagrosh** [comment]
Most of the DJ commands have secondary side-effects that you don't want non-DJs to have access to, even when alone. For example, `stop` also causes the bot to leave the voice channel. I think what you are asking for here is basically an alias that runs `remove all` and `skip`

**Reply by the-snesler** [reply]
I see, that makes sense. Though in my case, the intention was to make the bot leave :/
My friend was using the bot recently, and she couldn't figure out how to make it leave when she was done with it. Of course, it would've disconnected if left alone for long enough, but a certain other music bot that recently went offline also gave users full access when they were alone.
Your idea for an alias is super smart, though! I'll pass it on to the person running the instance we use. Thank you!

**Reply by jagrosh** [reply]
If you set an `alonetimeuntilstop` value in the config, your friend doesn't need to make it leave; it will automatically stop and leave after it is alone for enough time.

https://jmusicbot.com/config
